apple/internal/local_provisioning_profiles.bzl

```diff
@@ -0,0 +1,80 @@
+"""
+Bazel rules for using locally installed provisioning profiles for debug builds
+"""
+
+def _local_provisioning_profiles(repository_ctx):
+    repository_ctx.symlink(
+        "{}/Library/MobileDevice/Provisioning Profiles".format(repository_ctx.os.environ["HOME"]),
+        "profiles",
+    )
+
+    repository_ctx.file("BUILD", """
+filegroup(
+    name = "profiles",
+    srcs = glob(["profiles/*.mobileprovision"]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "empty",
+    srcs = [],
+    visibility = ["//visibility:public"],
+)
```

Comment:
Docs pending of course, but should the empty case work (allow the finder to find profiles) if no fallback_profiles are specified?

```bash
in (an implicit dependency) attribute of local_provisioning_profile rule //MyApp: '@LocalProfiles//:empty' does not produce any local_provisioning_profile $fallback_srcs files (expected .mobileprovision)
```

https://github.com/bazelbuild/rules_apple/pull/1190/files#diff-6af425aecce524a86c66a19c86dc87aca95abb271885b0aac0fac4801ba92d49R67

Reply:
Ah thanks I probably need to allow empty or something. I added this in this adaptation since we don't actually have this case so I didn't test it well enough